mne/gui/_coreg_gui.py

```diff
@@ -929,28 +924,42 @@ def _load_trans_fired(self):
         self.model.load_trans(trans_file)
 
     def _save_fired(self):
-        if self.n_scale_params:
-            subjects_dir = self.model.mri.subjects_dir
-            subject_from = self.model.mri.subject
-            subject_to = self.model.raw_subject or self.model.mri.subject
-        else:
-            subject_to = self.model.mri.subject
+        subjects_dir = self.model.mri.subjects_dir
+        subject_from = self.model.mri.subject
+
+        # check that fiducials are saved
+        skip_fiducials = False
+        if self.n_scale_params and not _find_fiducials_files(subject_from,
+                                                             subjects_dir):
+            msg = ("No fiducials have been saved for {src}. If fiducials "
```

Comment:
fiducials is intentionally plural because a fiducials file contains 3 fiducials. Would "No fiducials file has been found" be better?